diff --git a/include/data/cvector.h b/include/data/cvector.h
index eccc94a..df3938f 100644
--- a/include/data/cvector.h
+++ b/include/data/cvector.h
@@ -213,7 +213,7 @@ namespace lsp
                 return true;
             }
 
-            inline ssize_t do_index_of(const void *ptr) const
+            inline size_t do_index_of(const void *ptr) const
             {
                 for (size_t i=0; i<nItems; ++i)
                 {
@@ -412,7 +412,7 @@ namespace lsp
 
                 inline void take_from(cvector<T> *src) { do_take_from(src); }
 
-                inline ssize_t index_of(const T *item) const { return do_index_of(item); }
+                inline size_t index_of(const T *item) const { return do_index_of(item); }
 
                 inline bool copy_from(const cvector<T> *src) { return do_copy_from(src); }
         };
